app/src/main/res/layout/item_update_card.xml

```diff
@@ -40,15 +40,16 @@
       android:id="@+id/update_details_container"
       android:layout_width="match_parent"
       android:layout_height="wrap_content"
-      android:layout_below="@id/update_sequence">
+      android:layout_below="@id/update_sequence"
+      android:layout_marginBottom="@dimen/grid_3">
```

Comment:
I noticed the `update_backers_only` `TextView` is now off center because of this `grid_3` on the bottom.
<img width="283" alt="Screen Shot 2020-04-21 at 4 25 44 PM" src="https://user-images.githubusercontent.com/1289295/79910534-e6726a80-83ec-11ea-8841-4bf5678a778e.png">

Do we need it? How does it look without it?

Reply:
Fixed! :) 
<img width="368" alt="Screen Shot 2020-04-21 at 1 48 18 PM" src="https://user-images.githubusercontent.com/4083656/79912515-0139e480-83d7-11ea-9058-2b76ef6c7e5c.png">